opencodelists.org.conf.template

```diff
@@ -0,0 +1,10 @@
+server {
+    server_name opencodelists-proxy.opensafely.org;
+    listen ${PORT};
+
+    location /api/v1/dmd-mapping/ {
+        proxy_pass https://www.opencodelists.org;
+        proxy_ssl_server_name on;
```

Comment:
Ah, was this the missing piece from last time? I wonder if that was because the Github IPs we were proxying to already only serve single applications and so didn't need the server name in order to serve the correct certificate.

Reply:
It was!